x-pack/docs/en/security/operator-privileges/configure-operator-privileges.asciidoc

```diff
@@ -0,0 +1,61 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[configure-operator-privileges]]
+=== Configure operator privileges
+
+==== Enable operator privileges
+Operator privileges need to be enabled explicitly on each node in the cluster.
+This is done by configure the `elasticsearch.yml` file:
+
+[source,yaml]
+----------------------------
+xpack.security.operator_privileges.enabled: true
+----------------------------
+
+If the node is already running before the above change is made, you need restart
+it for the feature to take effect.
+
+WARNING: The feature needs to be either enabled or disabled consistently across all nodes
+in a cluster. Otherwise, you can get inconsistent behaviours depending on which node
+a request hits first and where it gets executed.
+
+==== Designate operator users
+Operator users are just normal {es} users with special privileges to perform
```

Comment:
I think "privileges" here, makes people think of RBAC privileges, so it might be better to introduce a different word while we're still explaining the concept.
Overall, "Operator privileges" is fine, but for someone learning about it for the first time, they need to change their thinking, and reusing words to mean something new, before they're got their head around the idea itself, can be confusing.

```suggestion
Operator users are just normal {es} users with special rights to perform
```